package.xml

```diff
@@ -28,6 +28,7 @@
   <depend>geographiclib</depend>
   <depend>message_filters</depend>
   <depend>nav_msgs</depend>
+  <depend>angles</depend>
```

Comment:
Don't you need to also added in the CMakelist?

Reply:
https://github.com/cra-ros-pkg/robot_localization/pull/747

Reply:
That's weird because project has compiled properly on bare `ros:galactic` image with rosdep. Probably another dependency has this library.